src/main/java/frc/robot/CompRobotContainer.java

```diff
@@ -36,6 +40,8 @@ public CompRobotContainer() {
         configureAutos();
 
         m_driveTrain.setDefaultCommand(getDriveCommand());
+
+        SmartDashboard.putNumber("percentage", 0);
```

Comment:
Please remove. Also, if this were actually used, you would want something like "led/percentage" as the key.